design-tokens/assets/android/StyleDictionary.kt

```diff
@@ -0,0 +1,19 @@
+
+
+// Do not edit directly
+// Generated on Fri, 20 Jan 2023 15:09:34 GMT
+
+
+
+package io.element.compound.tokens;
+
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.*
+
+object StyleDictionary {
+  /** The main brand color */
+  val brand = #0dbd8b
```

Comment:
This will not compile. The correct format would be:

```suggestion
val brand = Color(0xFF0dbd8b)
```